jwst/lib/set_telescope_pointing.py

```diff
@@ -684,6 +686,10 @@ def update_wcs_from_fgs_guiding(model, default_roll_ref=0.0, default_vparity=1,
         model.meta.wcsinfo.pc2_2
     ) = calc_rotation_matrix(roll_ref, np.deg2rad(v3i_yang), vparity=vparity)
 
+    # Set CRVAL as the guide star coordinates.
+    model.meta.wcsinfo.crval1 = model.meta.guidestar.gs_ra
+    model.meta.wcsinfo.crval2 = model.meta.guidestar.gs_dec
+
```

Comment:
So apparently the storing of these same values that happens in the block at https://github.com/spacetelescope/jwst/blob/a278351fc50a9336d07b7abc9b02f1eefe3f4ea9/jwst/lib/set_telescope_pointing.py#L2458 isn't sufficient for getting crvaln populated already?

Reply:
No. The guiding exposures take a completely different approach. I haven't tried to find the original documentation so I am not sure why; way to long ago for memory. I believe it is because guiding is "self-referential": It is reporting the information that is needed, so it already knows.  The TRACK algorithm, for regular science, is then based on this information to transform to the science detectors.  COARSE is the only one that needs the original observatory orientation, since guiding is not in use.